lib/utils/levenshteinDistance.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+"use strict";
+
+/**
+ * Compute levenshtein distance, see https://en.wikipedia.org/wiki/Levenshtein_distance for more info
+ * @param {string} firstStr first string
+ * @param {string} secondStr second string
+ * @param {number} [maxDistance=99] maximum distance, if distance is greater then returns -1
+ * @returns {number} computed distance
+ */
+module.exports = function levenshteinDistance(
```

Comment:
Did you copy this function from somewhere? If yes, it would be nice to add source to the comment inside code. So in future we could reference the source.

Reply:
No, I implemented it myself. I provided link to wiki page, I think it is enough